src/cmd/pkg/converter/terraform.go

```diff
@@ -0,0 +1,254 @@
+			}
+			tf.HCLID = tfid.SafeID(notifier.Slug)
+			tfOut.VictorOpsNotifiers = append(tfOut.VictorOpsNotifiers, tf)
+			notifierTypes[notifier.Slug] = "chronosphere_victorops_alert_notifier"
+		case notifier.Webhook != nil:
+			tf, err := chronosphere.WebhookAlertNotifierFromModel(&notifier)
+			if err != nil {
+				return nil, fmt.Errorf("error converting webhook notifier to Terraform: %v", err)
+			}
+			tf.HCLID = tfid.SafeID(notifier.Slug)
+			tfOut.WebhookNotifiers = append(tfOut.WebhookNotifiers, tf)
+			notifierTypes[notifier.Slug] = "chronosphere_webhook_alert_notifier"
+		default:
+			return nil, errors.New("notifier must set a target")
+		}
+	}
+
+	var err error
+	tfOut.NotificationPolicy, err = tfPolicy(alerts.NotificationPolicy, newNotifierTFIDGenerator(notifierTypes))
+	if err != nil {
+		return nil, fmt.Errorf("error converting notification policy to Terraform: %v", err)
+	}
+	tfOut.Monitors, err = sliceutil.MapErr(alerts.Monitors, tfMonitor)
+	if err != nil {
+		return nil, fmt.Errorf("error converting monitor to Terraform: %v", err)
+	}
+	tfOut.Collections, err = sliceutil.MapErr(alerts.Collections, tfCollection)
+	if err != nil {
+		return nil, fmt.Errorf("error converting collection to Terraform: %v", err)
+	}
+	tfOut.RecordingRules, err = sliceutil.MapErr(alerts.RecordingRules, tfRecordingRule)
+	if err != nil {
+		return nil, fmt.Errorf("error converting recording rule to Terraform: %v", err)
+	}
+	return tfOut, nil
+}
+
+func tfMonitor(in *models.Configv1Monitor) (*intschema.Monitor, error) {
+	b, err := in.MarshalBinary()
+	if err != nil {
+		return nil, err
+	}
+	var tf tfmodels.Configv1Monitor
+	err = tf.UnmarshalBinary(b)
+	if err != nil {
+		return nil, err
+	}
+	out, err := chronosphere.MonitorFromModel(&tf)
+	if err != nil {
+		return nil, err
+	}
+	out.HCLID = tfid.SafeID(tf.Slug)
+	// Convert tfid.ID fields to local references.
+	out.NotificationPolicyId = policyTFID(out.NotificationPolicyId)
+	out.CollectionId = collectionTFID(out.CollectionId)
+	return out, nil
+}
+
+func tfCollection(in *models.Configv1Collection) (*intschema.Collection, error) {
+	b, err := in.MarshalBinary()
+	if err != nil {
+		return nil, err
+	}
+	var tf tfmodels.Configv1Collection
+	err = tf.UnmarshalBinary(b)
+	if err != nil {
+		return nil, err
+	}
+	out, err := chronosphere.CollectionFromModel(&tf)
+	if err != nil {
+		return nil, err
+	}
+	out.HCLID = tfid.SafeID(tf.Slug)
+	out.NotificationPolicyId = policyTFID(out.NotificationPolicyId)
+	return out, nil
+}
+
+func tfRecordingRule(in *models.Configv1RecordingRule) (*intschema.RecordingRule, error) {
+	b, err := in.MarshalBinary()
+	if err != nil {
+		return nil, err
+	}
+	var tf tfmodels.Configv1RecordingRule
+	err = tf.UnmarshalBinary(b)
+	if err != nil {
+		return nil, err
+	}
+	out, err := chronosphere.RecordingRuleFromModel(&tf)
+	if err != nil {
+		return nil, err
+	}
+	out.HCLID = tfid.SafeID(tf.Slug)
+	out.ExecutionGroup = collectionTFID(out.ExecutionGroup)
```

Comment:
`BucketSlug` should use a `collectionTFID`, but `ExecutionGroup` is treated as a flat string, and generally isn't a reference